sdk-setup/src/mb2

```diff
@@ -225,7 +225,7 @@ COMMANDS
 
         -R    reverse apply patches (patch -R)
 
-    build [-p|--doprep] [-d|--enable-debug] [-j <n>] [<project-dir-or-file>]
+    build [-p|--doprep] [-d|--enable-debug] [-j <n>] [<project-dir-or-file>] [--nosign]
```

Comment:
I am not sure signing should be the default option. It requires some additional setup, we don't have supporting tools for key management, we miss guidelines etc. If we want to save users from typing extra chars, I would prefer addressing that by providing single letter option `[-s|--sign]` now.

The option should be listed before `<project-dir-or-file>`.